src/Facets/OpBNBBridgeFacet.sol

```diff
@@ -0,0 +1,196 @@
+            }
+            s.bridges[configs[i].assetId] = IL1StandardBridge(
+                configs[i].bridge
+            );
+        }
+
+        s.standardBridge = standardBridge;
+
+        emit OptimismInitialized(configs);
+    }
+
+    /// External Methods ///
+
+    /// @notice Register token and bridge
+    /// @param assetId Address of token
+    /// @param bridge Address of bridge for asset
+    function registerOpBNBBridge(address assetId, address bridge) external {
+        LibDiamond.enforceIsContractOwner();
+
+        Storage storage s = getStorage();
+
+        if (bridge == address(0)) {
+            revert InvalidConfig();
+        }
+
+        s.bridges[assetId] = IL1StandardBridge(bridge);
+
+        emit OptimismBridgeRegistered(assetId, bridge);
+    }
+
+    /// @notice Bridges tokens via OpBNB Bridge
+    /// @param _bridgeData Data contaning core information for bridging
+    /// @param _bridgeData Data specific to OpBNB Bridge
+    function startBridgeTokensViaOpBNBBridge(
+        ILiFi.BridgeData memory _bridgeData,
+        OpBNBData calldata _opBNBData
+    )
+        external
+        payable
+        nonReentrant
+        refundExcessNative(payable(msg.sender))
+        doesNotContainSourceSwaps(_bridgeData)
+        doesNotContainDestinationCalls(_bridgeData)
+        validateBridgeData(_bridgeData)
+    {
+        LibAsset.depositAsset(
+            _bridgeData.sendingAssetId,
+            _bridgeData.minAmount
+        );
+        _startBridge(_bridgeData, _opBNBData);
+    }
+
+    /// @notice Performs a swap before bridging via OpBNB Bridge
+    /// @param _bridgeData Data contaning core information for bridging
+    /// @param _swapData An array of swap related data for performing swaps before bridging
+    /// @param _bridgeData Data specific to OpBNB Bridge
+    function swapAndStartBridgeTokensViaOpBNBBridge(
+        ILiFi.BridgeData memory _bridgeData,
+        LibSwap.SwapData[] calldata _swapData,
+        OpBNBData calldata _opBNBData
+    )
+        external
+        payable
+        nonReentrant
+        refundExcessNative(payable(msg.sender))
+        containsSourceSwaps(_bridgeData)
+        doesNotContainDestinationCalls(_bridgeData)
+        validateBridgeData(_bridgeData)
+    {
+        _bridgeData.minAmount = _depositAndSwap(
+            _bridgeData.transactionId,
+            _bridgeData.minAmount,
+            _swapData,
+            payable(msg.sender)
+        );
+        _startBridge(_bridgeData, _opBNBData);
+    }
+
+    /// Private Methods ///
+
+    /// @dev Contains the business logic for the bridge via OpBNB Bridge
+    /// @param _bridgeData Data contaning core information for bridging
+    /// @param _bridgeData Data specific to OpBNB Bridge
+    function _startBridge(
+        ILiFi.BridgeData memory _bridgeData,
+        OpBNBData calldata _opBNBData
+    ) private {
+        Storage storage s = getStorage();
+        IL1StandardBridge nonStandardBridge = s.bridges[
```

Comment:
storing this in a variable seems unnecessary. 
Could integrate that into the next (conditional) statement

Reply:
Here storing in a var saves gas as we would need to read from storage twice otherwise.